tools/benchmark/metrics/prepare_metrics.sh

```diff
@@ -12,6 +12,18 @@ MYDIR=$(dirname $(realpath "$0"))
 main() {
   cd "${MYDIR}/../../../third_party"
   local zipurl
+  local repourl
+  for repourl in \
+    'https://github.com/veluca93/IQA-optimization.git'
+    'https://github.com/Netflix/vmaf.git'
+	  'https://github.com/thorfdbg/difftest_ng.git'
```

Comment:
The indentation for this line seems off.

Reply:
Something put a tab in there... oh well.